### ISSUE TITLE
Air Radar

### ISSUE THREAD

**Issue by Dale1994** [Bubbus/ACF-Missiles#51]
Seeing as targeting is prohibited on most ACF servers, why not create an ACF object that would help AA. It would have a find in cone, once pointed at a vehicle it will give out only: Direction, Velocity. 
This would help with simple lead computers and would NOT give out position data. Perhaps a ground version as well for tanks.

**Comment by Cre8or** [comment]
Why would this be necessary? Wiremod has lots of ways of achieving this. If the server you play on doesn't allow you to use guided missiles, or any targetting devices, then that's a problem on their end. Though I wonder why they even installed ACF Missiles in that case, as guided weapons are a big part of it.

**Comment by Bubbus** [comment]
This would be a pretty distant stretch goal if anything.  I don't get much time to develop this mod, so I have to focus on the most pressing needs - for now, that's countermeasures.

I'm not saying no - I'll accept any help I can get...  but maybe this is better suited to be in the main ACF if its primary use is for bullet lead computing?

**Comment by Dale1994** [comment]
It would be needed Cre8or for several reasons. If we aloud the use of E2, how can we trust that users would only gain information given above? With this we can allow people to make lead assisted systems that still require some one too aim. To sum this up, this would be a legal way to make lead based systems seeing as most servers I know of prohibit find() functions for E2 unless you own the objects.

**Comment by Cre8or** [comment]
The matter of the fact is that the process of targetting an entity is not our area of responsibility - this is where Wiremod comes into play, and where you have to come up with an idea on how you're going to handle your targetting system. If we were to add a replacement entity for everything that's blocked on server X or Y, we'd be stuck coding them forever.

My suggestion is that you look for alternative ways of locking onto things, perhaps with a ranger, or with a target finder and a beacon sensor to fetch the entity for you.

**Comment by Dale1994** [comment]
All entity discovery is disabled on GGG, Ive only managed to make ineffective rangerHull radars.
